config/routes.rb

```diff
@@ -6,6 +6,8 @@
   get '/account' => 'users#edit_current_user', as: :account
 
   resources :posts
+  resources :internal_posts, controller: 'posts', type: 'InternalPost'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.